pygments/lexers/c_cpp.py

```diff
@@ -19,6 +19,21 @@
 __all__ = ['CLexer', 'CppLexer']
 
 
+class CFamilyComments(RegexLexer):
```

Comment:
No need for a separate class; just do `using(this, state='comments')`, and add this as `'comments'` state in the `CFamily` lexer. That way, some things like subclassing are easier.

Reply:
I made this change (and the changes requested below). However now the tests are failing, with error:
`TypeError: get_tokens_unprocessed() got an unexpected keyword argument 'stack'.`

I suspect it is because the possible comment regex may receive no input (empty string). I cannot seem to match to the empty string in a sensible way. Any advice on how to resolve this would be greatly appreciated.